app/src/main/java/com/android/fundamentals/workshop02/Workshop2Fragment.kt

```diff
@@ -7,7 +7,7 @@ import androidx.fragment.app.Fragment
 import androidx.fragment.app.viewModels
 import com.android.fundamentals.R
 
-class Workshop2Fragment : Fragment(R.layout.fragment_workshop_1) {
+class Workshop2Fragment : Fragment(R.layout.fragment_workshop_2) {
```

Comment:
workshop 1 and 2 use same layout
1 and 2 workshop implement same but in different ways
Maybe just rename fragment_workshop_1 -> fragment_workshop_1_workshop_2 and use it?